Sources/SwiftInspectorCore/InitializerAnalyzer.swift

```diff
@@ -124,7 +129,25 @@ public struct InitializerStatement: Equatable {
 
   public struct Parameter: Equatable {
     public let name: String
-    public let typeName: String
+    public let typeNames: [String]
+
+    /// The most common use case is for a parameter to only contain one type
+    /// That's why we have this computed property as a convenience
+    public var typeName: String? {
+      return typeNames.first
+    }
+
+    public init(name: String, typeNames: [String]) {
+      self.name = name
+      self.typeNames = typeNames
+    }
+
+    /// The most common use case is for a parameter to only contain one type
+    /// That's why we have this convenience initializer
+    public init(name: String, typeName: String) {
```

Comment:
Do we use this one anywhere?

Reply:
I guess what I'm getting at is that it's not clear we should have if we never us it, and I wasn't seeing it.

Reply:
We are! All over the tests https://github.com/fdiaz/SwiftInspector/blob/f3e16254429cc08033b07157ac5274efe3b25d4e/Sources/SwiftInspectorCore/Tests/InitializerAnalyzerSpec.swift#L87

Reply:
Huh I wonder if these should be in an extension for tests?

Reply:
I still feel like this convenience makes sense. Most of the initializers have a single parameter.

Reply:
I guess when you think about _writing_ code this makes a lot of sense 🤔